integration/apps/minife/0005-Add-region-markup.patch

```diff
@@ -34,59 +34,20 @@ Subject: [PATCH] Adding geopm markup to CORAL version of miniFE.
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-Signed-off-by: Steve S. Sylvester <steve.s.sylvester@intel.com>
-Signed-off-by: Diana Guttman <diana.r.guttman@intel.com>
+Signed-off-by: Brad Geltz <brad.geltz@intel.com>
 ---
- src/Makefile                   | 14 ++++++++------
  src/SparseMatrix_functions.hpp | 24 ++++++++++++++++++++++++
  src/Vector_functions.hpp       | 24 ++++++++++++++++++++++++
- src/cg_solve.hpp               |  7 +++++++
- 4 files changed, 63 insertions(+), 6 deletions(-)
+ 2 files changed, 48 insertions(+)
 
-diff --git a/src/Makefile b/src/Makefile
-index 461bdde..0ef6065 100644
---- a/src/Makefile
-+++ b/src/Makefile
-@@ -16,7 +16,7 @@ MINIFE_MATRIX_TYPE = -DMINIFE_CSR_MATRIX
- 
- # BG/Q
- #CFLAGS = -DIBM_HPM -qlist -qsource  -g -O3 -qsmp=omp
--CFLAGS = -qlist -qsource  -g -O3 -qsmp=omp
-+CFLAGS = -g -O3 -fopenmp -I$(HOME)/build/geopm/include
- CXXFLAGS = $(CFLAGS)
- 
- # To enable memory high water mark reporting uncomment the
-@@ -25,15 +25,17 @@ CPPFLAGS = -I. -I../utils -I../fem $(MINIFE_TYPES) \
- 	$(MINIFE_MATRIX_TYPE) \
- 	-DHAVE_MPI -DMPICH_IGNORE_CXX_SEEK #-DMINIFE_REPORT_RUSAGE
- 
--LDFLAGS=$(CFLAGS)
--LIBS=
-+LDFLAGS=$(CFLAGS) -L$(HOME)/build/geopm/lib
-+LIBS=-lgeopm
- 
- # GNU CXX=mpicxx
- # GNU CC=mpicc
- 
- # BG/Q
--CXX=mpixlcxx_r
--CC=mpixlc_r
--
-+# CXX=mpixlcxx_r
-+# CC=mpixlc_r
-+#
-+CXX=mpicxx
```

Comment:
Should be $MPICC etc?

Reply:
Yeah, you're correct.  I was looking at Makefile.intel.openmp, and I was trying to use that instead of the BG/Q based regular Makefile.  But I just checked, and the binary that's built doesn't utilize MPI.  I'll fix this up.